app/forms/apo_form.rb

```diff
@@ -79,6 +79,14 @@ def collection_radio
     'none'
   end
 
+  def default_object_rights
+    return default_rights if model.nil?
+
+    default_object_rights_option = check_default_object_rights
```

Comment:
Do we need this allocation? Can you just move the call to the last line?

Reply:
Done